tests/component/test_watchdog_properties.py

```diff
@@ -0,0 +1,169 @@
+"""Tests for validating watchdog properties."""
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import Level, Edge
+from nidaqmx.errors import DaqError, DAQmxErrors
+from nidaqmx.system.watchdog import DOExpirationState
+
+
+def test__watchdog__get_boolean_property__returns_false(any_x_series_device):
+    """Test to validate getter for boolean property."""
+    with nidaqmx.system.WatchdogTask(any_x_series_device.name, timeout=0.5) as task:
+        task.start()
+
+        assert not task.expired
+        task.stop()
+
+
+@pytest.mark.parametrize("device_by_name", ["cDAQ1"], indirect=True)
+def test__watchdog__set_boolean_property__returns_assigned_value(device_by_name):
+    """Test to validate setter for boolean property."""
+    with nidaqmx.system.WatchdogTask(device_by_name.name, timeout=0.5) as task:
+
+        task.expir_trig_trig_on_network_conn_loss = True
+
+        assert task.expir_trig_trig_on_network_conn_loss
+
+
+@pytest.mark.parametrize("device_by_name", ["cDAQ1"], indirect=True)
+def test__watchdog__reset_boolean_property__returns_default_value(device_by_name):
+    """Test to validate reset for boolean property."""
+
+    with nidaqmx.system.WatchdogTask(device_by_name.name, timeout=0.5) as task:     
+        task.expir_trig_trig_on_network_conn_loss = True
+
+        del task.expir_trig_trig_on_network_conn_loss
+
+        assert not task.expir_trig_trig_on_network_conn_loss
+
+
+def test__watchdog__task_not_running_get_expired_property___throws_daqerror(any_x_series_device):
+    """Test to validate error case for boolean property."""
+    with nidaqmx.system.WatchdogTask(any_x_series_device.name, timeout=0.5) as task:
+        try:
+            _ = task.expired
+        except DaqError as e:
+            assert e.error_type == DAQmxErrors.CANNOT_GET_PROPERTY_WHEN_TASK_NOT_RESERVED_COMMITTED_OR_RUNNING
```

Comment:
This will pass if no exception is thrown. I overlooked this in some of the previous tests. 

I think the right solution is [pytest.raises](https://docs.pytest.org/en/4.6.x/reference.html#pytest-raises)

You can use `with pytest.raises(DaqError) as exc_info:` and then validate `exc_info.value.error_type` in the assert section.

Reply:
Fix for test where I overlooked this: https://github.com/ni/nidaqmx-python/pull/259

Reply:
Updated to `pytest.raises`.